### PR TITLE
Update bcftools filter cmd

### PR DESCRIPTION
**Pull request by Takadonet**
Update bcftools filter command so it will be compliant for bcftools version 1.9+

Had to update test for rearrange_snv_matrix.pl due to changes of sorting by default alphabetically with leaf node with same value.

### CODE REVIEW

**Review by apetkau — CHANGES_REQUESTED** [review]
This is awesome @Takadonet. Thanks so much for updating everything. I got all the tests to pass.

A few comments:

1. Can you update the `CHANGELOG.md` to add your specific changes (should it be release `1.8.2`. or `1.9`?)
2. Since this is meant to update the code to work with **bcftools** `1.9+`, could you update the `install_deps.sh` script to install this version of bcftools/other libraries? And could you update the `README.md` file so it does not mention version `1.5` anymore?

File: README.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ cd snvphyl-tools
 ./install_deps.sh
 #wait patiently to completion and setup the following environment variables
 #example below
-export PATH=/home/test/snvphyl-tools/bcftools-1.5:/home/test/snvphyl-tools/MUMmer3.23:$PATH
-export BCFTOOLS_PLUGINS=/home/test//snvphyl-tools/bcftools-1.5/plugins
+export PATH=/home/test/snvphyl-tools/bcftools-1.9:/home/test/snvphyl-tools/MUMmer3.23:$PATH
+export BCFTOOLS_PLUGINS=/home/test//snvphyl-tools/bcftools-1.9/plugins
 PERL5LIB=/home/test/snvphyl-tools/lib/perl5
```
Comment (apetkau) [inline README.md]: I think this needs to be `export PERL5LIB` here.

**Review by apetkau — CHANGES_REQUESTED** [review]
Thanks @Takadonet this looks great. However, I was having an issue with the tests. The script `install_deps.sh` does not install `samtools`, which was causing issues.

I believe we should either add this to `install_deps.sh`, or just mention in the `README.md` file to install `samtools` version 1.9.

File: README.md (hunk 1 above)
Comment (Takadonet) [reply README.md]: Fixed

**Review by apetkau — APPROVED** [review]

**Comment by apetkau** [comment]
Looks good. Thanks @Takadonet

**Pull request merged**